### PR TITLE
Update permission map

### PR DESCRIPTION
**Pull request by cgzones**
Add watch permissions, added in https://github.com/SELinuxProject/selinux-kernel/commit/ac5656d8a4cdd93cd2c74355ed12e5617817e0e7

Add `perf_event` class, added in https://github.com/SELinuxProject/selinux-kernel/commit/da97e18458fb42d7c00fac5fd1c56a3896ec666e

Add `lockdown` class, added in https://github.com/SELinuxProject/selinux-kernel/commit/59438b46471ae6cdfb761afc8c9beaf1e428a331

Add `anon_inode` class, added in https://github.com/SELinuxProject/selinux-kernel/commit/29cd6591ab6fee3125ea5c1bf350f5013bc615e1

Weight removed permission with value 10, see https://github.com/SELinuxProject/selinux-kernel/commit/42a9699a9fa179c0054ea3cf5ad3cc67104a6162

Drop inexistent `permissions execute_no_trans` and `entrypoint` from `chr_file`

Signed-off-by: Christian Göttsche <cgzones@googlemail.com>

### CODE REVIEW

File: setools/perm_map [hunk 1]
```diff
@@ -146,13 +146,18 @@ class dir 26
            relabelto         w        10
              mounton         b         1
              quotaon         b         1
-              swapon         b         1
+              swapon         b        10
```
Comment (pebenito) [inline setools/perm_map:149]: 2 thoughts here.

1. Why would `swapon` be different from `mounton`?
2. `swapon` isn't checked, so arguably it should be `n 1`

File: setools/perm_map [hunk 2]
```diff
@@ -828,17 +861,18 @@ class db_blob 10
              setattr         w         7
            relabelto         w         1
 
-class filesystem 10
+class filesystem 11
            associate         n         1
             quotaget         r         1
          relabelfrom         r        10
-          transition         w         1
+          transition         w        10
```
Comment (pebenito) [inline setools/perm_map:868]: this is not checked, so it should arguably be `n 1` too.

File: setools/perm_map (hunk 1 above)
Comment (cgzones) [reply setools/perm_map]: Added commit setting all removed permissions to `n 1`

**Comment by pebenito** [comment]
After further investigating I found these permissions were last checked (some were never checked) in kernel 2.6.30. The last kernel that had these permissions defined was 4.1 in 2015. I think it's safe to delete the unchecked permissions instead.

**Comment by pebenito** [comment]
linter crashed. This PR doesn't change code.

**Pull request merged**